server/README.md

```diff
@@ -164,6 +164,14 @@ To set the welcome theme, use the following environment value :
 To add a custom provider extend the Keycloak image and add the provider to the `/opt/jboss/keycloak/standalone/deployments/`
 directory.
 
+## Running custom scripts on startup
+
+To run a custom `.cli` script on container startup extend the Keycloak image and add the `.cli` file to the
+`/opt/jboss/keycloak/tools/docker-entrypoint.d` directory (create it if necessary).
```

Comment:
Change name of the directory. Something like "/opt/jboss/startup-scripts". "tools" shouldn't be there and it definitively should not contain the name "docker".

Reply:
Why definitely not the name docker? The name `docker-entrypoint.d` is the community standard as I've shown in my previous comments.

Reply:
There are quite a few arguments why we don't like to include "docker" in the description...

One of them is that we want to make Keycloak as friendly as possible for the Kubernetes-based environments (such as OpenShift). Since Kubernetes can work without Docker (using Podman), it creates a sort of conflict in our minds. Not to mention, that you can build Keycloak image without Docker as well (using Buildah). 

So I agree with @stianst here, please pick some more neutral name. Using `/opt/jboss/startup-scripts` seems to be a good start.

Reply:
Perhaps the `docker-entrypoint.sh` (already existing, part of the reason I chose this name) should also be renamed?

Reply:
I've relocated the directory to `/opt/jboss/startup-scripts`.

Reply:
Very good point! I've created [KEYCLOAK-9609](https://issues.jboss.org/browse/KEYCLOAK-9609) to replace `docker-entrypoint.sh` with better name. Thanks for pointing this out!